src/controllers/order.confirmation.controller.ts

```diff
@@ -65,21 +65,29 @@ export const render = async (req: Request, res: Response, next: NextFunction) =>
 
         const checkout = (await getCheckout(accessToken, orderId)).resource as Checkout;
 
-        // required to capture order type in matomo
-        if (!basketLinks.data.enrolled && (itemType === undefined || itemType === "")) {
-            const item = checkout?.items?.[0];
-            return res.redirect(getWhitelistedReturnToURL(req.originalUrl) + getItemTypeUrlParam(item));
-        }
-        else if (basketLinks.data.enrolled && req.query.itemTypes === undefined) {
+        const originalUrl = req.originalUrl;
+        const separator = originalUrl.includes('?') ? '&' : '?';
+
+        const redirectWithItemTypes = (itemTypes: string) => {
+            const redirectUrl = getWhitelistedReturnToURL(originalUrl) + separator + itemTypes;
+            return res.redirect(redirectUrl);
+        };
+
+        // Required to capture order type in matomo
+        if (!basketLinks.data.enrolled && (itemType == undefined || itemType === "")) {
```

Comment:
the strict equality here has been removed, was this intentional? shouldn't it still be 
if (!basketLinks.data.enrolled && (itemType === undefined || itemType === "")) {